backend/dataall/modules/shares_base/db/share_object_repositories.py

```diff
@@ -195,56 +171,330 @@ def get_all_share_items_in_share(session, share_uri, status=None, healthStatus=N
             )
         )
```

Comment:
small change, status is now a list because we use this function in other places in the code instead of list_shareable_items